website/pages/payments.html

```diff
@@ -1,22 +1,60 @@
+<style>
+    #bottomNotes {
+        display: block;
+        padding-left: 18px;
+        padding-right: 18px;
+        padding-bottom: 18px;
+    }
+    #topPool {
+        padding-top: 18px;
+        padding-left: 18px;
+        padding-right: 18px;
+    }
+    #topPool > div > div > svg {
+        display: block;
+        height: 280px;
+    }
+    .poolWrapper {
+        border: solid 1px #c7c7c7;
+        border-radius: 5px;
+        padding: 5px;
+        margin-bottom: 18px;
+    }
+    .poolLabel {
+        font-size: 1.2em;
+        text-align: center;
+        padding: 4px;
+    }
+    .poolMinerTable {
+
+    }
+    table {
+        width: 100%;
+    }
+</style>
+
 {{ for(var pool in it.stats.pools) { }}
-<table>
-    <thead>
-    <tr>
-        <th>Blocks</th>
-        <th>Time</th>
-        <th>Miners</th>
-        <th>Shares</th>
-        <th>Amount</th>
-    </tr>
-    </thead>
-    {{ for(var p in it.stats.pools[pool].payments) { }}
-    <tr>
-        <td>{{=it.stats.pools[pool].payments[p].blocks}}</td>
-        <td>{{=it.stats.pools[pool].payments[p].time}}</td>
-        <td>{{=it.stats.pools[pool].payments[p].miners}}</td>
-        <td>{{=Math.round(it.stats.pools[pool].payments[p].shares)}}</td>
-        <td>{{=it.stats.pools[pool].payments[p].paid}} {{=it.stats.pools[pool].symbol}}</td>
-    </tr>
-    {{ } }}
-</table>
+
+<div class="poolMinerTable">
+    <table class="pure-table">
+      <thead>
+      <tr>
+          <th>Blocks</th>
+          <th>Transaction</th>
+          <th>Miners</th>
+          <th>Shares</th>
+          <th>Amount</th>
+      </tr>
+      </thead>
+      {{ for(var p in it.stats.pools[pool].payments) { }}
+      <tr>
+          <td>{{=it.stats.pools[pool].payments[p].blocks}}</td>
+          <td><a target="_blank" href="http://explorer.myhush.org/tx/{{=it.stats.pools[pool].payments[p].txid}}">{{=it.stats.pools[pool].payments[p].txid}}</td>
```

Comment:
alright, will nix the link and come up with a var or something to add yourself later